scripts/collectVsixes.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
```

Comment:
I also wrote this script which helps to find and copy the vsixes from the mono repo. This will need to get re-worked path-wise once moved to the monorepo itself, along with some of the test cases that rely on path building and navigation.